projects/igniteui-angular/src/lib/grids/grid/grid-filtering-ui.spec.ts

```diff
@@ -3421,6 +3421,34 @@ describe('IgxGrid - Filtering Row UI actions #grid', () => {
             tick(100);
             expect(chip.componentInstance.selected).toBeTruthy();
     }));
+
+    it('Should close filterRow when changing filterMode from \'quickFilter\' to \'excelStyleFilter\'', (async () => {
```

Comment:
Add verification for grid body height in the test.

Reply:
We cannot reproduce the bug in the test, because we call fix.detectChanges() which always fixes the issue in test. This makes the grid's body height check not applicable.